### DISCUSSION TITLE
Add a user to an existing group

### DISCUSSION THREAD

**Discussion by mistoll** [wixtoolset/issues#7432] [Questions]
I want to create a user and add it to an existing group (users).

Here's what I have so far:
```
<util:QueryWindowsWellKnownSIDs />
<util:Group Id="GRP_Users" Name="[WIX_ACCOUNT_USERS]"/>

<DirectoryRef Id="mydir">
  <Component Id="myUser" Guid="...">
    <RegistryValue Id="myUserKey" Root="HKLM" Key="[RK_Root]\Users\[PlayerUserName]" Type="integer" Name="Installed" Value="1" />
    <util:User Id="myUtilUser" CreateUser="yes" FailIfExists="no" UpdateIfExists="yes" LogonAsService="no" PasswordNeverExpires="yes" CanNotChangePassword="yes" Name="[UserName]" Password="[USERPASSWORD]" RemoveOnUninstall="no" Disabled="no">
      <util:GroupRef Id="GRP_Users" />
    </util:User>
  </Component>
</DirectoryRef>
```

This has been asked a few times
[1] https://stackoverflow.com/questions/69180391/add-user-to-a-group-in-a-localized-msi-installer
[2] https://wix-users.narkive.com/ht5w39UW/add-user-to-predefined-group-fails

We used to solve this by using a custom action. And that's what Rob Mensching suggested in [2], if i get him right. But that was back in 2008. But I wonder if there's a solution without having to implement/maintain our own custom action.

The problem with the code above is pretty much what is discussed in [1]. util:Group expects the Name attribute to be "Users", but WIX_ACCOUNT_USERS is "BUILTIN\Users".

UPDATE:
The code for QueryWindowsWellKnownSIDs  is here: https://github.com/wixtoolset/Util.wixext/blob/8a8a25695351ee542f08886a9d0957c78c6af366/src/ca/OsInfo.cpp#L354 and there is already WIX_ACCOUNT_PERFLOGUSERS_NODOMAIN which is WIX_ACCOUNT_PERFLOGUSERS without the domain prefix. Is adding more "_NODOMAIN" variants an option?

**Comment by robmen** [comment]
Extending the existing `_NODOMAIN` pattern seems reasonable given the use case.

**Reply by mistoll** [reply]
Here's a PR: https://github.com/wixtoolset/wix/pull/407
